.travis.yml

```diff
@@ -1,23 +1,56 @@
 # This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
 # THIS FILE IS BEING USED TO CI-TEST THE CI SCRIPT ITSELF - do not use this for your moveit repo but instead use the version in README.md
-sudo: required
-dist: trusty
+os: linux
+dist: bionic
 services:
   - docker
 language: cpp
 cache: ccache
 compiler: gcc
 
 notifications:
-  email:
+  email: true
 
 env:
   global: # default values
-    - ROS_DISTRO=eloquent
+    - ROS_DISTRO=foxy
     - ROS_REPO=ros
     - SCRIPT=travis.sh
-  matrix:
-    - SCRIPT=unit_tests.sh
+
+jobs:
```

Comment:
Shouldn't https://github.com/ros-planning/moveit_ci/pull/122/commits/7b11a06dfe4954c205c94bbe5946944b39d47f12 be merged into master first?

Reply:
It is.